doc/reference/reference_lua/datetime.rst

```diff
@@ -1,62 +1,958 @@
+            ---
+            - 2021-08-26T00:00:00+0300
+            - +7 days
+            ...
+
+            tarantool> dt:add(iv)
+            ---
+            - 2021-09-02T00:00:00+0300
+            ...
+
+            tarantool> dt:add{ day = 7 }
+            ---
+            - 2021-09-09T00:00:00+0300
+            ...
+
+        ..  _datetime-add-example2:
+
+        **Example #2:**
+
+        ..  code-block:: tarantoolsession
+
+            tarantool> dt = datetime.new {
+                        day = 29,
+                        month = 2,
+                        year = 2020
+                        }
+            ---
+            ...
+
+            tarantool> dt:add{month = 1, adjust = 'none'}
+            ---
+            - 2020-03-29T00:00:00Z
+            ...
+
+            tarantool> dt = datetime.new {
+                        day = 29,
+                        month = 2,
+                        year = 2020
+                        }
+            ---
+            ...
+
+            tarantool> dt:add{month = 1, adjust = 'last'}
+            ---
+            - 2020-03-31T00:00:00Z
+            ...
+
+            tarantool> dt = datetime.new {
+                        day = 31,
+                        month = 1,
+                        year = 2020
+                        }
+            ---
+            ...
+
+            tarantool> dt:add{month = 1, adjust = 'excess'}
+            ---
+            - 2020-03-02T00:00:00Z
+            ...
+
+    ..  _datetime-sub:
+
+    ..  method:: sub( { input[, adjust ] } )
+
+        Modify an existing datetime object by subtracting values of the input argument.
+
+        :param table input: an :ref:`interval object <interval_obj>` or an equivalent table (see **Example**)
+        :param string adjust: defines how to round days in a month after an arithmetic operation.
+                                Possible values: ``none``, ``last``, ``excess``. Defaults to ``none``.
+                                The logic is similar to the one of the ``:add()`` method -- see :ref:`Example #2 <datetime-add-example2>`.
+
+        :return: datetime_object
+        :rtype: cdata
+
+        **Example:**
+
+        ..  code-block:: tarantoolsession
+
+            tarantool> dt = datetime.new {
+                        day = 26,
+                        month = 8,
+                        year = 2021,
+                        tzoffset  = 180
+                        }
+            ---
+            ...
+
+            tarantool> iv = datetime.interval.new {day = 5}
+            ---
+            ...
+
+            tarantool> dt, iv
+            ---
+            - 2021-08-26T00:00:00+0300
+            - +5 days
+            ...
+
+            tarantool> dt:sub(iv)
+            ---
+            - 2021-08-21T00:00:00+0300
+            ...
+
+            tarantool> dt:sub{ day = 1 }
+            ---
+            - 2021-08-20T00:00:00+0300
+            ...
+
+
+.. _interval_obj:
+
+interval_object
```

Comment:
It probably should be a title, but it doesn't look like one, is a correct markup is used?

![screen-1722866922](https://github.com/user-attachments/assets/9341521e-2094-4d62-a92e-bcedbd2c20de)

Reply:
It's a second-level topic title, looks OK.